pubspec.yaml

```diff
@@ -29,6 +29,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.0
+  flutter_bloc: ^6.1.1
```

Comment:
Why did you add this dependency? Where do you need to use that package?

Reply:
I don't need it. I tried to use it in a solution that I had in mind but in the end I went the other way. I removed the dependency.